### PR TITLE
Allow pasting of features in attribute table

### LINKED ISSUE #600: Allow pasting of features in attribute table
https://github.com/qgis/QGIS/commit/499c102b57aa562f5d4c86a9960c05a43b1cce76
Merge: eaeea21 9b34202
Author: Nathan Woodrow
Date:   Wed May 20 20:46:10 2015 +1000

    Merge pull request #2047 from alexbruy/attrtable-paste

### PR DESCRIPTION
**Pull request by yjacolin**
* Fix #600
* Change shorcut to lowercase

### CODE REVIEW

**Comment by yjacolin** [comment]
@SrNetoChan are you ok to do a PR to my branch?

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 1]
```diff
@@ -989,23 +989,25 @@ The other buttons at the top of the attribute table window provide the
...
-* |mActionInvertSelection| :sup:`Invert selection` (also with :kbd:`Ctrl+R`)
+  and to enable functionalities described below (also with :kbd:`Ctrl+e`)
+* |mActionSaveEdits| :sup:`Save Edits` (also with :kbd:`Ctrl+s`)
+* |mActionUnselectAttributes| :sup:`Unselect all` (also with :kbd:`Ctrl+u`)
+* |mActionSelectedToTop| :sup:`Move selected to top` (also with :kbd:`Ctrl+t`)
+* |mActionInvertSelection| :sup:`Invert selection` (also with :kbd:`Ctrl+r`)
 * |mActionCopySelected| :sup:`Copy selected rows to clipboard` (also with
-  :kbd:`Ctrl+C`)
+  :kbd:`Ctrl+c`)
+* |mActionPasteSelected| :sup:`Paste from clipboard to a new row` (also with
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: It should be [|mActionEditPaste|](http://docs.qgis.org/testing/en/docs/documentation_guidelines/substitutions.html#attribute-table). I know it's not consistent with the previous icon naming but this is how the icon is [originally named](https://github.com/qgis/QGIS/blob/master/images/themes/default/mActionEditPaste.png) and I think we should be tied to this (for updating process). There are also a mActionEditCopy and a mActionEditCut icons.

Note that there are two icons used in QGIS application to copy features: 

* 
mactioneditcopy
[mActionEditCopy](https://github.com/qgis/QGIS/blob/master/images/themes/default/mActionEditCopy.png) icon in Digitizing toolbar, Edit menu and Identify Results panel
* and mactioncopyselected [mActionCopySelected](https://github.com/qgis/QGIS/blob/master/images/themes/default/mActionCopySelected.png) in attribute table and ...

I think we should rather have an unique icon and [already asked](http://osgeo-org.1560.x6.nabble.com/Two-different-icons-for-same-action-td5238491.html) in the developer list which icon to keep but got no answer.

**Comment by DelazJ** [comment]
Is it better to use lower-case letters? I use to see these shortcuts with upper-case letters (I assume user knows that he just uses the letter and not its upper-cased variant)

**Comment by yjacolin** [comment]
In my opinion, yes because some application use shortcuts with upper case and so you need to use the shift button.

**Comment by SrNetoChan** [comment]
@yjacolin Sorry, I have missed this discussion until now. What do you need from me?

**Comment by yjacolin** [comment]
@SrNetoChan could you update my branch to add the missing screenshot (mActionPasteSelected)
@DelazJ which one do think is better to use selected or not?

**Comment by DelazJ** [comment]
@SrNetoChan following comment in #600, there's a need to update the table attribute screenshot

@yjacolin currently there's already a substitution available in the conf.py file for the "Paste features" icon in attribute table and it's |mActionEditPaste|. 
If you are asking for the icon to use in QGIS application, i don't know; i'd say the one with yellow background (for consistency with paste icon), renamed in mActionEditCopy (or move all to ...Selected naming, even though selection is implicit to cut/copy/paste processes)

**Comment by SrNetoChan** [comment]
@yjacolin I will take care fo the #600 Screenshot.
I'm not sure if my git skills are enough to directly update your branch, but I will try.

**Comment by yjacolin** [comment]
@SrNetoChan As @DelazJ said, there is already a PastIcon. so I switch to this one rather to create a new one. I will merge the Pr as soon as test are finished.

**Pull request merged**